documentation/blog/2023-09-20-runes.md

```diff
@@ -115,6 +120,7 @@ This works, but it's pretty weird! We've found that the store API can get rather
 With runes, things get much simpler:
```

Comment:
If we want to mention `.svelte.js` in prose, it should be somewhere around here instead.

Reply:
How about now?